tests/spread/integration/zip/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Integration tests for zip
+
+execute: |
+  rootfs="$(install-slices zip_bins)"
+  
+  chroot "${rootfs}/" zip -h2
+  chroot "${rootfs}/" zipcloak -h
+  chroot "${rootfs}/" zipnote -h
+  chroot "${rootfs}/" zipsplit -h
+
+  echo "Hello, World" > "${rootfs}/testfile.txt"
+  chroot "${rootfs}" zip testfile.zip testfile.txt
+  chroot "${rootfs}" zipnote testfile.zip > "${rootfs}/foo.tmp"
+  echo -e "@ This is a test comment" >> "${rootfs}/foo.tmp"
+  chroot "${rootfs}" zipnote -w testfile.zip < "${rootfs}/foo.tmp"
```

Comment:
Since it appears `foo.tmp` is created via shell redirection, I think we can just do this:
```shell
  chroot "${rootfs}" zipnote testfile.zip > foo.tmp
  echo -e "@ This is a test comment" >> foo.tmp
  chroot "${rootfs}" zipnote -w testfile.zip < foo.tmp
```